data/binding_affinity/PDBbind/pdbbind_test.csv

```diff
@@ -0,0 +1,15 @@
+pdb,label
```

Comment:
We can probably put this file in a dedicated tests folder. Can try to set something up with `pytest` this week.